projects/hslayers/src/components/save-map/layman-utils.ts

```diff
@@ -86,3 +90,14 @@ export function isAtLeastVersions(ep: HsEndpoint, version: string) {
 
   return true;
 }
+
+/**
+ * Wait until layer synchronization is complete
+ * @param layer - OL Layer
+ */
+export async function awaitLayerSync(layer: Layer): Promise<any> {
```

Comment:
```suggestion
export async function awaitLayerSync(layer: Layer): Promise<boolean> {
```

Not sure which type restriction would work best here. Might be `<true | undefined>`? Also removing the return type and let the TS compiler to infer the type is OK here I guess.

Reply:
Boolean is fine I think